### PR TITLE
Recounts `rise_number` and related variables every once in a while

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Every 5 minutes, the rise and ascend thresholds are recalculated, to account for people leaving the round / joining late

## Why It's Good For The Game
This helps a lot when a lot of people leave the game, or a lot of people latejoin

## Images of changes
image

**This image should say `-1` but I accidentally closed my instance before taking a screenshot**
image

## Testing
See above

<hr>

## Changelog

:cl:
fix: Cult will now be a little more consistent when a lot of people latejoin / leave the round early
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/team_cult.dm [hunk 1]
```diff
@@ -31,6 +31,17 @@ RESTRICT_TYPE(/datum/team/cult)
...
+	/// Timer until we do a recount of cultist members
+	var/recount_timer
+
+/datum/team/cult/New(list/starting_members)
+	. = ..()
+	recount_timer = addtimer(CALLBACK(src, PROC_REF(cult_threshold_check)), 5 MINUTES, TIMER_STOPPABLE|TIMER_DELETE_ME|TIMER_LOOP)
+
+/datum/team/cult/Destroy(force, ...)
+	. = ..()
+	QDEL_NULL(recount_timer)
```
Comment (Contrabang) [inline code/modules/antagonists/cult/team_cult.dm:43]:
```suggestion
	deltimer(recount_timer)
	return ..()
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/cult/team_cult.dm (hunk 1 above)
Comment (DGamerL) [reply code/modules/antagonists/cult/team_cult.dm:43]: Oops

**Review by Contrabang — CHANGES_REQUESTED** [review]
can ya remove `addtimer(CALLBACK(src, PROC_REF(cult_threshold_check)), 2 MINUTES) // Check again in 2 minutes for latejoiners` please?

**Comment by DGamerL** [comment]
> can ya remove `addtimer(CALLBACK(src, PROC_REF(cult_threshold_check)), 2 MINUTES) // Check again in 2 minutes for latejoiners` please?

Done!

**Review by Burzah — APPROVED** [review]

**Pull request merged**